dashboards-reports/public/components/report_definitions/report_settings/time_range.tsx

```diff
@@ -203,6 +203,15 @@ export function TimeRangeSelect(props) {
     setIsLoading(false);
   };
 
+  const commonlyUsedRanges = uiSettings!
```

Comment:
@iget-master thanks for the PR. We recently added support for `dateFormat` in uiSettings. There is a module (not a prop since some code will run in discover) to expose the ui settings client https://github.com/opensearch-project/dashboards-reports/blob/93d885c641c8c7fc8beeb081bee6ffd30b44f73b/dashboards-reports/public/components/utils/settings_service.ts#L16-L23

I believe the only change needed is here, and uiSettings can be called this way
https://github.com/opensearch-project/dashboards-reports/blob/93d885c641c8c7fc8beeb081bee6ffd30b44f73b/dashboards-reports/public/components/main/main_utils.tsx#L199

Could you update the PR please? thanks

Reply:
@iget-master any thoughts?

Reply:
I think that's ok. I'm a little busy next days, but asap i can update the pr